clients/render/index.js

```diff
@@ -62,8 +62,9 @@ domReady(function () {
 
 
 		jQuery(document).on('cf.ajax.request', (event, obj) => {
+
 			//do not run if component is empty
-			if( ! theComponent ){
+			if( theComponent.props.fieldsToControl.length <= 0 ){
```

Comment:
@New0 When I tested, the problem was `theComponent` was null. 

If you want to test `theComponent.props.fieldsToControl.length` fine, but first test that props and props.fieldsControl are properties and theComponent is an object.